api/v1alpha1/dnshealthcheckprobe_types.go

```diff
@@ -104,7 +103,7 @@ func (p *DNSHealthCheckProbe) Default() {
 }
 
 func (p *DNSHealthCheckProbe) ToString() string {
-	return fmt.Sprintf("%v://%v:%v/%v", p.Spec.Protocol, p.Spec.Hostname, p.Spec.Port, p.Spec.Path)
+	return fmt.Sprintf("%v://%v:%v%v", p.Spec.Protocol, p.Spec.Hostname, p.Spec.Port, p.Spec.Path)
```

Comment:
so here we will end up with "https://host:0/path" I think?

Reply:
we will always end up with a valid address here. It even checks for the `path` to start with `/` or `?`. So something like `https://host:80health` won't be allowed.